### PR TITLE
Add possibility to publish labelled markers

### PR DESCRIPTION
**Pull request by aikolina**
- Add the possibility to publish labelled markers
- Update main README
- Minor code style clean-up

### CODE REVIEW

**Review by randaz81 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,64 +1,90 @@
...
+  ```bash
+  yarpdev --device transformServer --ROS::enable_ros_publisher 0 --ROS::enable_ros_subscriber 0
+  ```
+- Open a third terminal and start a `yarp-vicon-bridge` device:
+  ```bash
+  yarpdev --device vicon --hostname "192.168.10.2"
+  ```
+  :warning: the default `hostname` is `192.168.10.2` (see [Vicon Network Cards Settings](https://docs.vicon.com/display/Connect/Configuring+network+card+settings#Configuringnetworkcardsettings-IPAddresses)) but some network test might be needed to find out the port where the Vicon output is actually streamed.
+
+### Using the Vicon setup and R1
```
Comment (randaz81) [inline README.md:66]: It is not clear to me why R1 is mentioned here.
roscore can run on any machine with ROS installed.
yarpserver --ros can run on any machine with ROS installed.
R1 is not needed at all..

**Review by randaz81 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (aikolina) [reply README.md:66]: Actually those ones were the original instructions available on the repository, and I didn't test myself on R1. So I might have misinterpreted them... 😅

File: src/YarpViconBridge.cpp [hunk 2]
```diff
@@ -84,17 +84,17 @@ bool YarpViconBridge::open(Searchable &config) {
...
     if(config.check("publish_labeled_markers"))
     {
-        publish_labeled_markers=config.find("publish_labeled_markers").asInt()==true;
+        publish_labeled_markers=config.find("publish_labeled_markers").asBool()==true;
     }
 
     if(config.check("publish_unlabeled_markers"))
     {
-        publish_unlabeled_markers=config.find("publish_unlabeled_markers").asInt()==true;
+        publish_unlabeled_markers=config.find("publish_unlabeled_markers").asBool()==true;
```
Comment (Nicogene) [inline src/YarpViconBridge.cpp]: `==true` it can be removed

**Review by Nicogene — CHANGES_REQUESTED** [review]

File: src/YarpViconBridge.cpp (hunk 2 above)
Comment (aikolina) [reply src/YarpViconBridge.cpp]: ok (-__-)>

**Comment by Nicogene** [comment]
Merged, thanks!

**Pull request merged**